basxconnect/mailer_integration/mailchimp/datasource.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+
+import mailchimp_marketing
+from django.conf import settings
+
+from basxconnect.mailer_integration.abstract import abstract_datasource
+from basxconnect.mailer_integration.abstract.abstract_mailer_person import MailerPerson
+from basxconnect.mailer_integration.mailchimp.mailchimp_person import MailchimpPerson
+
+URBANMOSAIC_ALL_MEMBERS_LIST_ID = "4606fb0179"
+SWISS_SEGMENT_ID = "36705"
+
+
+class MailchimpDatasource(abstract_datasource.Datasource):
+    def __init__(self) -> None:
+        self.client = mailchimp_marketing.Client()
+        self.client.set_config({"api_key": settings.MAILCHIMP_API_KEY, "server": "us5"})
+
+    def get_persons(self) -> List[MailerPerson]:
+        segment = self.client.lists.get_segment_members_list(
+            list_id=URBANMOSAIC_ALL_MEMBERS_LIST_ID,
+            segment_id=SWISS_SEGMENT_ID,
+            count=1000,
```

Comment:
Maybe this is a stupid question, but what happens if we have a list a with more than 1000 contacts? Does this still get synchronized correctly?

Reply:
Nope. At the moment 1000 is the limit. Urbanmosaic CH has 200 contacts currently.
I created a github issue (https://github.com/basxsoftwareassociation/basxconnect/issues/138). I think the whole synchronization logic might change anyway, so I don't want to over-engineer it just now. But I agree we should not forget about this limitation.

Reply:
Could we put the count number in a setting, like MAILCHIMP_MAX_SYNC_COUNT or similar? That way the limitation is expressed a bit more directly and it would lift the hardcoded-constraint to be a server-config, in case of an "emergency".